packages/shared/src/components/ProfilePicture.tsx

```diff
@@ -120,4 +120,31 @@ function ProfilePictureComponent(
   );
 }
 
-export const ProfilePicture = forwardRef(ProfilePictureComponent);
+const Group: React.FC<{
+  children:
+    | React.ReactElement<ProfilePictureProps>[]
+    | React.ReactElement<ProfilePictureProps>;
+  limit?: number;
+  total?: number;
+}> = ({ children, limit = 3, total }) => {
+  const childrenMap = React.Children.toArray(children).slice(0, limit);
+
+  return (
+    <div className="flex">
+      {childrenMap.map(
```

Comment:
Do note it should also have that +32 component, which is not a profile picture now.
I do like the approach here.

Reply:
good point, that's what the `limit` and `total` props are for 👌